### ISSUE TITLE
!null and Exp() : Internal error: !NULL destructor called.

### ISSUE THREAD

**Issue by alaingdl** [gnudatalanguage/gdl#1140]
Exp(!null) is broken, but Gsl_Exp(!null) not :smile: 
```
GDL> print, exp(!null)
Internal error: !NULL destructor called.
Save your work and restart GDL (GDL is still functional, but !NULL will not work anymore).
Please report at https://github.com/gnudatalanguage/gdl/issues
% EXP: Variable is undefined: !NULL
% Execution halted at: $MAIN$          
```

(sorry not to look at that, but I would like to finish the changes I am working on)

**Comment by GillesDuvert** [comment]
please label "NULL" every issue related to the somewhat messy introduction of !NULL in GDL code...

**Comment by GillesDuvert** [comment]
closed by #1174
